agent/ecscni/plugin_windows.go

```diff
@@ -65,7 +98,20 @@ func (client *cniClient) setupNS(ctx context.Context, cfg *Config) (*current.Res
 
 	seelog.Debugf("[ECSCNI] Completed setting up the container namespace: %s", cfg.ContainerID)
 
-	return nil, nil
+	if _, err := ecsBridgeResult.GetAsVersion(currentCNISpec); err != nil {
+		seelog.Warnf("[ECSCNI] Unable to convert result to spec version %s; error: %v; result is of version: %s",
```

Comment:
Nit, could you use the structured log wrapper here
e.g: https://github.com/aws/amazon-ecs-agent/blob/2874131a5f3d1c0908e0f2ebad62e48b8ff5850f/agent/engine/task_manager.go#L242-L245